pkgs/applications/editors/vim/configurable.nix

```diff
@@ -154,7 +154,11 @@ in stdenv.mkDerivation rec {
   '' + stdenv.lib.optionalString stdenv.isLinux ''
     patchelf --set-rpath \
       "$(patchelf --print-rpath $out/bin/vim):${stdenv.lib.makeLibraryPath buildInputs}" \
-      "$out"/bin/{vim,gvim}
+      "$out"/bin/vim
+  '' + stdenv.lib.optionalString (guiSupport != "false") ''
```

Comment:
If gui support is enabled, the condition will be true, and then we do need to run that patchelf command.